ui/pages/home/home.container.js

```diff
@@ -155,50 +167,68 @@ const mapStateToProps = (state) => {
     newTokensImported: getNewTokensImported(state),
     newNetworkAddedConfigurationId: appState.newNetworkAddedConfigurationId,
     onboardedInThisUISession: appState.onboardedInThisUISession,
+    ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+    waitForConfirmDeepLinkDialog: getWaitForConfirmDeepLinkDialog(state),
+    institutionalConnectRequests,
+    modalOpen: state.appState.modal.open,
+    mmiPortfolioUrl: getMmiPortfolioUrl(state),
+    mmiPortfolioEnabled: getMmiPortfolioEnabled(state),
+    notificationsToShow: getSortedAnnouncementsToShow(state).length > 0,
+    ///: END:ONLY_INCLUDE_IN
   };
 };
 
-const mapDispatchToProps = (dispatch) => ({
-  closeNotificationPopup: () => closeNotificationPopup(),
-  ///: BEGIN:ONLY_INCLUDE_IN(snaps)
-  removeSnapError: async (id) => await removeSnapError(id),
+const mapDispatchToProps = (dispatch) => {
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  const mmiActions = mmiActionsFactory();
```

Comment:
I have seen this factory and asked a question about it elsewhere.... but i can't remember the response so if this is deja vu feel free to link me to my original comment lol. ..why does this need to be a factory?

Reply:
It's because all the actions share the same method structure, the only thing different is the action name, so in order to not repeat a lot of code, we've used a factory.